packages/scene-composer/package.json

```diff
@@ -64,7 +64,11 @@
     "copy:notice": "cp ../../NOTICE NOTICE",
     "prepack": "npm run copy:license && npm run copy:notice",
     "pack": "npm pack",
-    "svglint": "svglint src/assets/**/*.svg"
+    "svglint": "svglint src/assets/**/*.svg",
+    "test:ui": "npx playwright test --update-snapshots",
+    "test:ui:reliability": "npx playwright test --repeat-each 5 --workers=1",
```

Comment:
Repeating each test 5 times to ensure there are no flakey tests. Passing `--workers=1` ensures that the tests are serialized rather than run in parallel. There was a [known issue](https://github.com/microsoft/playwright/issues/12827) with running tests in parallel that is likely related. Tests run without `--repeat-each` flag are consistently successful & can run in parallel.